scheduler/task_finder_test.go

```diff
@@ -371,6 +382,27 @@ func TestCompareTaskRunnersWithStaticTasks(t *testing.T) {
 				Activated: true,
 				Project:   "disabled",
 			},
+			task.Task{
+				Id:        "bar",
+				Status:    evergreen.TaskUndispatched,
+				Activated: true,
+				Requester: evergreen.PatchVersionRequester,
+				Project:   "patching-disabled",
+			},
+			task.Task{
+				Id:        "baz",
+				Status:    evergreen.TaskUndispatched,
+				Activated: true,
+				Requester: evergreen.GithubPRRequester,
+				Project:   "patching-disabled",
+			},
+			task.Task{
+				Id:        "runnable",
+				Status:    evergreen.TaskUndispatched,
+				Activated: true,
+				Requester: evergreen.RepotrackerVersionRequester,
+				Project:   "patching-disabled",
+			},
```

Comment:
Where are the asserts that go with these?

Reply:
The asserts are that all methods provide the same results, validated against each others.